src/CHANGES.txt

```diff
@@ -18,6 +18,7 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
   From Daniel Moody:
     - Change the default for AppendENVPath to delete_existing=0, so path
       order will not be changed, unless explicitly set (Issue #3276)
+    - fix bug where no msvc's are found and an empty list is indexed (PR# #3288)
```

Comment:
`Fix bug causing error on windows when no MSVC installation is available.  `

No need to self reference the PR. Only reference an Issue if one's been filed.

Sorry to be a bit nitpicky about this but CHANGES.txt is where people often look to see if their problem has been fixed.

Reply:
remove PR reference in 6a61fae